docs/contributing.md

```diff
@@ -125,7 +130,7 @@ Valid component names are listed in the [__label
 list__](https://github.com/rucio/rucio/labels) and are usually specified on the
 issue of the change.
 
-If you use the default commit message template, make sure you edit it.
+Make sure to summarize the `WHY` in the commit message.
```

Comment:
Hmm maybe something different here:

If possible, add additional explanations concerning the commit, such as motivation for certain decisions, in the body of the commit. 

?

Reply:
I changed the text a bit, also added a reference to this blog-post: https://cbea.ms/git-commit/
It is commonly known and references the most important rules.